.cicd/platforms/pinned/centos-7.7-pinned.dockerfile

```diff
@@ -17,9 +17,10 @@ RUN curl -s https://packagecloud.io/install/repositories/rabbitmq/erlang/script.
 RUN curl -s https://packagecloud.io/install/repositories/rabbitmq/rabbitmq-server/script.rpm.sh | bash && \
     yum install -y rabbitmq-server
 # upgrade pip installation
-RUN . /opt/rh/rh-python36/enable && \
-    pip install --upgrade pip
-# build cmake
+RUN source /opt/rh/rh-python36/enable && \
+    pip install --upgrade pip && python -m pip install requests && \
+    python -m pip install requests_unixsocket
```

Comment:
This can just be
pip3 install requests requests_unixsocket

Reply:
I used `pip` rather than `pip3` as I noticed that `pip` is used in `Centos 7.7` docker file for installing python modules. So I follow the same pattern.